# NavFn x RPP: changes against /opt/ros/jazzy/share/nav2_bringup/params/nav2_params.yaml
# written by parity/scripts/generate_matrix_configs.py; 1 key(s) changed
# controller_server.FollowPath.plugin: 'nav2_mppi_controller::MPPIController' -> 'nav2_regulated_pure_pursuit_controller::RegulatedPurePursuitController'  (plugin name)
--- installed/nav2_params.yaml
+++ NavFn_RPP/params.yaml
@@ -196,7 +196,7 @@
       iteration_count: 1
       model_dt: 0.05
       motion_model: DiffDrive
-      plugin: nav2_mppi_controller::MPPIController
+      plugin: nav2_regulated_pure_pursuit_controller::RegulatedPurePursuitController
       prune_distance: 1.7
       regenerate_noises: true
       temperature: 0.3
